components/direct-order/form/steps/ReviewOrder.vue

```diff
@@ -76,7 +76,38 @@
               <span>${{ (deliveryCost / 100).toLocaleString() }}</span>
             </div>
             <div class="flex justify-between mt-2">
-              <span>Tip</span>
+              <div>
+                <span class="mr-4">
+                  Tip
+                </span>
+                <button
+                  class="tip-button"
+                  :class="{active: tipPercentEquals(10)}"
+                  @click="onTipButtonClicked(10, $event)"
+                >
+                  10%
+                </button>
+                <button
+                  class="tip-button"
```

Comment:
@yassinya i really like how you refactored this with v-for and iterated instead of repeating code when you redid it!